byron/ledger/executable-spec/src/Byron/Spec/Ledger/Core.hs

```diff
@@ -36,7 +36,7 @@ import           Data.Word (Word64, Word8)
 import           GHC.Generics (Generic)
 import           Numeric.Natural (Natural)
 
-import           Cardano.Prelude (NoUnexpectedThunks(..))
+import           NoThunks.Class (NoThunks(..))
```

Comment:
I don't know what the import grouping convention is, but I would put it with the import group above it, not with the other Cardano imports.